src/poddefaults/poddefault-example-minio.yaml.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: kubeflow.org/v1alpha1
+kind: PodDefault
+metadata:
+ name: access-minio
+spec:
+ desc: Allow access to Minio
+ selector:
+   matchLabels:
+     access-minio: "true"
+ env:
+   - name: AWS_ACCESS_KEY_ID
+     valueFrom:
+       secretKeyRef:
+         name: mlpipeline-minio-artifact
+         key: accesskey
+         optional: false
+   - name: AWS_SECRET_ACCESS_KEY
+     valueFrom:
+       secretKeyRef:
+         name: mlpipeline-minio-artifact
+         key: secretkey
+         optional: false
```

Comment:
These point to pipeline's secret that has minio's credentials.  Don't you create your own?  Why not point to that

Reply:
Oh wait, the secret you create is named the same as what pipelines creates.  How will that work? If pipelines is installed, which secret will be in every user's namespace, yours or pipelines'?  What if they relate to different minios?

I think you need to name your secret for minio credentials to something unique to you, then cite that secret here.  Probably something that includes the app_name so you don't collide with others, too